components/DateTimePicker.js

```diff
@@ -13,19 +13,40 @@ import { View } from './View';
 
 const isIos = Platform.OS === 'ios';
 
+const MODES = { DATE: 'date', DATETIME: 'datetime', TIME: 'time' };
```

Comment:
export this for outside usage / manipulation

Reply:
👍
Will rename to `DATEPICKER_MODES` for better readability when importing